automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainGroupApiSpec.scala

```diff
@@ -33,11 +37,15 @@ class AuthDomainGroupApiSpec extends FreeSpec with Matchers with WorkspaceFixtur
           withCleanBillingProject(user) { projectName =>
             withWorkspace(projectName, "AuthDomainGroupApiSpec_workspace", Set(authDomainName)) { workspaceName =>
 
-              // user is one of the authdomain group members
-              groupNameToMembersEmails(authDomainName) should contain (user.email)
-              // user can access workspace and see the authdomain group
-              AuthDomainMatcher.checkVisibleAndAccessible(projectName, workspaceName, List(authDomainName))(user.makeAuthToken())
+              eventually {
+                // user is one of the authdomain group members
+                groupNameToMembersEmails(authDomainName) should contain(user.email)
```

Comment:
this should be separated so that the call to `groupNameToMembersEmails(authDomainName)` is inside the `eventually` and the check to see if the response contains the email is outside the eventually. This makes it easier, in case of test failure, to easily figure out if the issue is with the call or the check.